odoo80/scripts/library.sh

```diff
@@ -93,7 +93,7 @@ function collect_pip_dependencies(){
 
     pip install ${PIP_OPTS} ${DEPENDENCIES}
 
-    for REQ in $( find ${TEMPDIR} -type f -iname "requirements.txt" ); do
+    for REQ in $( find ${TEMPDIR} -maxdepth 1 -type f -iname "requirements.txt" ); do
         /usr/share/vx-docker-internal/odoo80/gen_pip_deps ${REQ} ${DEPENDENCIES_FILE}
```

Comment:
@JesusZapata are you 100% sure that all the requirements will always be in the first level?

Reply:
@ruiztulio 
Yes, in the case of the "odoo" the file "requirements.txt" is in the root 

![image](https://user-images.githubusercontent.com/1387970/30714188-ea4c5aa2-9edf-11e7-9f9c-5fc36b13c36f.png)

This fix is necessary because using the original code found two requirements.txt
requirements.txt and doc/requirements.txt but the file doc/requirements.txt contains the requirement "mercurial" this package no have python3 version

If you try to install mercurial on python3 show the follow error

![image](https://user-images.githubusercontent.com/1387970/30714030-5c1122b8-9edf-11e7-966d-7d7ad314f815.png)

On the other hand the dependency of the documentation is not required, IMHO

Reply:
@JesusZapata if all the other repos used here have the dependencies in the first level only is ok, not just odoo, that's why I akes if you are 100% sure

Reply:
Are you 100% sure?

Reply:
Yes, I made all test and all test ran well.